core/templates/dev/head/pages/exploration_editor/translation_tab/translator_overview_directive.html

```diff
@@ -14,6 +14,7 @@
     <div class="progress oppia-translation-progess-bar">
       <div class="progress-bar progress-bar-success" role="progressbar" ng-style="getTranslationProgressStyle()"></div>
     </div>
+    <span class="protractor-test-translation-NumericalStatus">(<[numberOfRequiredAudio-numberOfAudioNotAvailable]>/<[numberOfRequiredAudio]>)</span>
```

Comment:
Make it `protractor-test-translation-numerical-status`.

Reply:
Done